sympy/polys/polyroots.py

```diff
@@ -861,6 +862,31 @@ def roots(f, *gens,
     >>> roots([1, 0, -1])
     {-1: 1, 1: 1}
 
+    ``roots`` will only return roots expressible in radicals.
+    If the given polynomial has some or all of its roots inexpressible in radicals,
+    the result of ``roots`` will be incomplete or empty respectively.
+
+    Example where result is incomplete:
+
+    >>> roots((x-1)*(x**5-x+1), x)
+    {1: 1}
```

Comment:
You should clarify a bit why this is incomplete e.g. "This polynomial has an unsolvable quintic factor whose roots cannot be expressed in radicals. The factor ``(x - 1)`` has a rational root which is returned since ``roots`` always finds all rational roots."

The example below could have some explanation as well.